flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -252,3 +252,32 @@ def mach(self) -> pd.PositiveFloat:
 # pylint: disable=fixme
 # TODO: AutomotiveCondition
 OperatingConditionTypes = Union[GenericReferenceCondition, AerospaceCondition]
+
+
+def create_operating_condition_from_mach_reynolds(
```

Comment:
the name is a bit lengthy, maybe is better:
```suggestion
def operating_condition_from_mach_reynolds(
```

Reply:
name is now shorter